components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/SQLConstants.java

```diff
@@ -359,6 +359,16 @@ public class SQLConstants {
             "UM_ORG_ATTRIBUTE.UM_ATTRIBUTE_VALUE FROM UM_ORG INNER JOIN UM_ORG_ATTRIBUTE ON UM_ORG.UM_ID " +
             "= UM_ORG_ATTRIBUTE.UM_ORG_ID INNER JOIN (%s) AS FILTERED_ORG ON UM_ORG.UM_ID = FILTERED_ORG.UM_ID;";
 
+    public static final String GET_ORGANIZATIONS_META_ATTRIBUTES = "SELECT DISTINCT " +
```

Comment:
Don't we need to provide queries that works in all supported DB types?

Reply:
This is now updated with separate queries for Oracle and MSSQL. And currently tested for the following Databases:
- H2
- MSSQL
- Mysql
- PostgreSQL
- DB2